quarkchain/evm/transactions.py

```diff
@@ -153,8 +165,20 @@ def sign(self, key, network_id=None):
         key = normalize_key(key)
 
         self._in_mutable_context = True
-        self.v, self.r, self.s = ecsign(self.hash_unsigned, key)
-        self.version = 0
+
+        if self.version == 0:
+            self.v, self.r, self.s = ecsign(self.hash_unsigned, key)
+        if self.version == 1:
```

Comment:
will this change break previous tx?

Reply:
no, this only use for tests, and the default value